core/metrics/impl/metrics_watcher.cpp

```diff
@@ -60,7 +60,7 @@ namespace kagome::metrics {
     }
   }
 
-  outcome::result<size_t> MetricsWatcher::measure_storage_size() {
+  outcome::result<uintmax_t> MetricsWatcher::measure_storage_size() {
```

Comment:
what was wrong with size_t?

Reply:
size_t is not guaranteed to be large enough to contain file_size() result.